RedHat-UX/red-hat-design-system
# chore: update deps

## What I did

1. update deps, including glob 9
2. update glob() call sites to match
3. remove unused `open` dep
4. update pfe deps. Closes #801 

## Testing Instructions

1. check the DP, the footer inline demos, the nested demos. In particular, check the DP demos for the components which had their CSS linted in https://github.com/RedHat-UX/red-hat-design-system/pull/799/commits/a42d2bb6f900db1ae7b9bee1146ec252b01b7c0a

## Notes to Reviewers
